lib/lotus/model/migration.rb

```diff
@@ -0,0 +1,277 @@
+require 'sequel'
+require 'sequel/extensions/migration'
+require 'logger'
+
+module Lotus
+  module Model
+    # Allow users to easily group schema changes and migrate the
+    # database to a newer version or revert to a previous version.
+    #
+    # A migration class must be subclass of Lotus::Model::Migration
+    # and must have 2 instance methods:
+    #
+    #   * up   - to migrate
+    #   * down - to rollback
+    #
+    # Both instance methods could access following schema modification methods:
+    #
+    #   * add_column
+    #   * add_index
+    #   * create_view
+    #   * drop_column
+    #   * drop_index
+    #   * drop_table
+    #   * drop_view
+    #   * rename_table
+    #   * rename_column
+    #   * set_column_default
+    #   * set_column_type
+    #
+    # For more details, please consult documentation
+    # at http://sequel.jeremyevans.net/rdoc/files/doc/schema_modification_rdoc.html
+    #
+    # @example
+    #   require 'lotus/model/migration'
+    #
+    #   class MyMigration < Lotus::Model::Migration
+    #     def up
+    #       create_table(:authors) do
+    #         primary_key :id
+    #         String :name
+    #       end
+    #     end
+    #
+    #     def down
+    #       drop_table(:authors)
+    #     end
+    #   end
+    #
+    #   # to migrate
+    #   MyMigration.apply(DB, :up)
```

Comment:
@joneslee85 This comment puzzles me. :smiley_cat: `DB` derives from Sequel, but what's the meaning here in Lotus::Model?